### DISCUSSION TITLE
External Storage in AIO causes Nextcloud to freeze

### DISCUSSION THREAD

**Discussion by BigRrred** [nextcloud/all-in-one#1062] [Questions] [answered]
Forgive my lack of technical terminology etc, I'm more an enthusiast than a full-fledged software engineer. 
Since the most recent update to the Nextcloud AIO v1.7.1 (hosted on Ubunutu 20.04LTS), Nextcloud has begun freezing due to high IO wait times freezing the CPU. I know it's directly related to the 'External Storage' app because if immediately after the VM starts up, I disable External Storage, then the system runs fine. The AIO application is hosted on Proxmox - when I try to access files on a SMB share (TrueNAS) through nextcloud, the IO Delay (IO Wait time reported by Netdata) goes from less than 1% average, to over 40%. Is anyone else having this issue? If so, can it be easily fixed? My files are safe, stored externally on TrueNAS, however I don't want to lose all the data in apps such as calendar, chat, deck, etc by creating a new AIO instance. I have tested the connection to TrueNAS by connecting to the SMB share directly on a separate device, and on a separate Nextcloud VM (linuxserver.io docker) and they don't have issues so I'm fairly confident that it's not related to the file server. These issues started almost immediately AFTER the most recent updates to Nextcloud AIO.

**Comment by szaimen** [comment]
This may be caused by https://github.com/nextcloud/server/issues/30282

**Comment by BigRrred** [comment, marked as answer]
Update * Seems like it was a hardware issue with the SSD it was running off. I transferred the VM to a new Samsung 870 Evo, with nothing else on it and now it's running perfectly. High IO wait seemed to be caused by the CPU waiting for the failing SSD to respond.

**Reply by szaimen** [reply]
Nice, glad that you solved it!
